src/arx7.js

```diff
@@ -1,11 +1,12 @@
 import debug from 'debug';
 import irc from 'irc';
-import config from '../config';
+import config from '../config.json';
```

Comment:
With Node, you don't need to specify the `.json` - it works automatically :smile: